x-pack/plugins/security_solution/cypress/integration/alerts_detection_rules_custom.spec.ts

```diff
@@ -215,8 +215,7 @@ describe('Custom detection rules creation', () => {
   });
 });
 
-// FLAKY: https://github.com/elastic/kibana/issues/83772
-describe.skip('Custom detection rules deletion and edition', () => {
+describe('Custom detection rules deletion and edition', () => {
```

Comment:
Was this flakiness the result of the ML tag issue and `editFirstRule()` fetching an ML rule?